scRNA-seq/02-filtering_scRNA.Rmd

```diff
@@ -84,13 +84,14 @@ coldata <- data.frame(files = alevin_file,
 ```
 
 Using the `coldata` data frame that we set up, we can now run the `tximeta()` to import our expression data while automatically finding and associating the transcript annotations that were used when we performed the quantification.
+We'll also tell `tximeta()` to only read in the barcodes that `alevinQC` decided passed quality control.
 
 The first time you run `tximeta()` you may get a message about storing downloaded transcriptome data in a cache directory so that it can retrieve the data more quickly the next time.
 We recommend you use the cache, and accept the default location.
 
 ```{r read_data, live = TRUE}
 # Read in alevin results with tximeta
-bladder_sce <- tximeta(coldata, type = "alevin")
+bladder_sce <- tximeta(coldata, type = "alevin", alevinArgs = list(filterBarcodes=TRUE))
```

Comment:
Updating this means that we need to go through the notebook to find all the places where we talk about the number of cells and update those to the correct new values.

I can see right away that line 106 will need an update, but I suspect it is more extensive than that!

Minor style: we have been using spaces around `=` everywhere, so we will want that here, and may want to break this onto multiple lines